<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="arqCSV">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Insira arquivos .CSV</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="arqSelect">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="textoTituloTabela">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Titulo da tabela</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="titulo">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>271</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Arquivos convertidos</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tabela convertida</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="tabConvert">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>381</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Estilo da Tabela</string>
    </property>
    <widget class="QRadioButton" name="radio_tab">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>106</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Tabela</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rd_quad">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>106</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Quadro</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="arqxlsx">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Insira arquivos .xlsx</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>441</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
